docs/man_pages/project/configuration/edit-configuration.md

```diff
@@ -28,7 +28,7 @@ WARNING: This command is not applicable to mobile website projects. To view the
 Command | Description
 ----------|----------
 [mobileframework](mobileframework.html) | Lists all supported versions of the current development framework.
-[mobileframework](mobileframework.html) | Sets the selected development framework version for the project.
+[mobileframework&nbsp;set](mobileframework-set.html) | Sets the selected development framework version for the project.
```

Comment:
is there a particular reason to use `&nbsp`, `prop add`, `prop print`, etc. are not using it